_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,118 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to every `<meta name="viewport">` elements with a `content` attribute in a document.
+
+## Expectation
+
+The `content` attribute of the test target: 
+- do not have the property `user-scalable` or it is set to `yes`
+- do not have the property `maximum-scale` 
+
+## Assumptions
+
+- There may be some inconsistencies with how different browser vendors interpret `<meta name="viewport">`.
```

Comment:
I think this should be in the Accessibility Support section, not in the Assumptions